app/pages/lab/subject-set.cjsx

```diff
@@ -65,6 +65,9 @@ SubjectSetListingTable = React.createClass
 SubjectSetListing = React.createClass
   displayName: 'SubjectSetListing'
 
+  contextTypes:
+    user: React.PropTypes.object
```

Comment:
Recently a few problems emerged which were caused by `context` not playing ball nicely, so I would stop using it. `user` is available as prop in the tree, so why not just use that instead? You could add it to `<SubjectSetListing ... user={@props.user}/>` `<SubjectViewer subject={subject} user={@props.user} />`.

Reply:
Makes sense. I initially chose context because of ease of addition, but I'm definitely up to make the change if passing down props is more reliable. Will push up the adjustment shortly.